main/src/com/google/refine/grel/ControlFunctionRegistry.java

```diff
@@ -306,7 +306,8 @@ static public void registerControl(String name, Control c) {
         registerFunction("combin", new Combin());
         registerFunction("degrees", new Degrees());
         registerFunction("radians", new Radians());
-        registerFunction("randomNumber", new RandomNumber());
+        registerFunction("random", new RandomNumber());
+        registerFunction("randint", new RandomNumber());
```

Comment:
I do not think it is very useful to register the function as `randint`, as the `int` part specifies that it should return random *integers*, and that is not the case for this function.